modules/core/petra/petra-sql-dsl-api/src/main/java/com/liferay/petra/sql/dsl/expression/ScalarDSLQueryAlias.java

```diff
@@ -10,14 +10,8 @@
 /**
```

Comment:
Change commit message to `LPS-204574 Make DefaultScalarDSLQueryAlias extends Alias`